### DISCUSSION TITLE
High CPU consumption on start

### DISCUSSION THREAD

**Discussion by boromyr** [mark-wiemer/ahkpp#582] [Q&A]
When I open my ahk work folder, the extension consumes a lot of CPU and takes a long time to index the files.
![image](https://github.com/user-attachments/assets/960adb41-37b5-440b-8d64-a72b73ab49e8)

**Comment by mark-wiemer** [comment]
Sorry about this. Can you share further logs? Please follow https://github.com/mark-wiemer-org/ahkpp/blob/main/docs/Troubleshooting.md#debug-logs

Thanks!
